.env.development

```diff
@@ -1,3 +1,3 @@
 VUE_APP_MODE = development
-VUE_APP_BACKEND_PREFIX = http://localhost:5000
+VUE_APP_BACKEND_PREFIX = http://localhost:8080
```

Comment:
Strange that the HTTP API is running on a different port as the SignalR one. Perhaps a typo?